### PR TITLE
deps: upgrade to bref 1.7.14 & php 8.2 extensions readyness

### PR DESCRIPTION
**Pull request by shouze**
@Nyholm, related to following PRs that leads tu publishing of php 8.2.0 official bref image.
- https://github.com/brefphp/bref/pull/1343
- https://github.com/brefphp/bref/pull/1344
- https://github.com/brefphp/bref/pull/1346

### CODE REVIEW

**Review by Nyholm — APPROVED** [review]
Just waiting on ci to be green

File: layers/xdebug/Dockerfile [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-    ./configure; \
-    make; \
-    make install; \
+  cd /tmp; \
+  git clone --depth=1 https://github.com/xdebug/xdebug.git; \
+  cd xdebug; \
+  phpize; \
+  ./configure; \
+  make; \
+  make install; \
```
Comment (mnapoli) [inline layers/xdebug/Dockerfile:15]: Formatting lost here, that might make it harder to maintain. Also xdebug compatible with 8.2 was released, so the if could actually be removed?

Comment (shouze) [reply layers/xdebug/Dockerfile:15]: @mnapoli good catch!
I can also use in place the new heredoc docker feature :wink: 
Let me a moment I will push that.

Comment (shouze) [reply layers/xdebug/Dockerfile]: @mnapoli done in https://github.com/brefphp/extra-php-extensions/pull/376/commits/0541f3336563be3df6d78a815cf9bb096ba1ca7a

Comment (mnapoli) [reply layers/xdebug/Dockerfile]: Oh nice feature, I love it!

But also there's a stable xdebug release for PHP 8.2, so maybe we don't want to compile from master anymore?

Comment (shouze) [reply layers/xdebug/Dockerfile]: Yup, let's :fire: that

Comment (shouze) [reply layers/xdebug/Dockerfile]: done in https://github.com/brefphp/extra-php-extensions/pull/376/commits/97c834bd2b321ca1c6a46c72520ff57c8118a20c

Comment (shouze) [reply layers/xdebug/Dockerfile]: Looks Good To Merge I think, can't wait to use php 8.2 on our app with Bref!

**Comment by Nyholm** [comment]
Thank you

**Pull request merged**